dependent/plastic-and-reconstructive-surgery-global-open.csl

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/npg -->
+  <info>
+    <title>Plastic and Reconstructive Surgery. Global open.</title>
```

Comment:
Let's change the title to "Plastic and Reconstructive Surgery - Global Open". I try to standardize on using hyphens as much as possible.